### PR TITLE
Update release notes:

### PR DESCRIPTION
**Pull request by nizarq**
Included a small description of cross component correlation changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 2.3.0-beta1
+- Added the ability to correlate http request made between different components represented by different application insights resources. This feeds into the impoved [application map experience](http://aka.ms/AiAppMapPreview).
```
Comment (cijothomas) [inline CHANGELOG.md]: typo- imp**r**oved

Comment (nizarq) [inline CHANGELOG.md]: thanks - fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 2.3.0-beta1
+- Added the ability to correlate http request made between different components represented by different application insights resources. This feeds into the improved [application map experience](http://aka.ms/AiAppMapPreview).
```
Comment (cijothomas) [inline CHANGELOG.md]: @SergeyKanzhelev - you have added exception statistics right? That should be added here as well.

Comment (SergeyKanzhelev) [inline CHANGELOG.md]: It's disabled by default. I'll add it when we will enable it in the default configuration

Comment (cijothomas) [inline CHANGELOG.md]: ok

**Pull request merged**